docs/src/pages/NativeRatingPage/NativeRatingPage.js

```diff
@@ -0,0 +1,116 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import React from 'react';
+
+import readme from '../../../../backpack-react-native/packages/react-native-bpk-component-rating/README.md';
+import androidScreenshotDefault from '../../../../backpack-react-native/packages/react-native-bpk-component-rating/screenshots/android/default.png';
+import androidScreenshotSizes from '../../../../backpack-react-native/packages/react-native-bpk-component-rating/screenshots/android/sizes.png';
+import androidScreenshotVertical from '../../../../backpack-react-native/packages/react-native-bpk-component-rating/screenshots/android/vertical.png';
+// import iosScreenshotDefault from '../../../../backpack-react-native/packages/react-native-bpk-component-rating/screenshots/ios/default.png';
```

Comment:
Is there a reason this is commented out?

Reply:
I mentioned in the PR description:

> I've commented out the iOS stuff for now because I can't get the RN codebase running to take the screenshots. Working on it!

Reply:
Can we remove it now and then make another PR to add or will it be done in this PR?

Reply:
Yeah I'm gonna update it before I merge. I figured it was ready for review now though as it won't change much

Reply:
All updated now.